generator/explores/explore.py

```diff
@@ -39,3 +45,50 @@ def get_view_lookml(self, view: str) -> dict:
         if self.views_path is not None:
             return lkml.load((self.views_path / f"{view}.view.lkml").read_text())
         raise Exception("Missing view path for get_view_lookml")
+
+    def _get_default_channel(self, view: str) -> Optional[str]:
+        channel_params = [
+            param
+            for _view_defn in self.get_view_lookml(view)["views"]
+            for param in _view_defn.get("parameters", [])
+            if _view_defn["name"] == view and param["name"] == "channel"
```

Comment:
I wonder if this pattern should be abstracted out at some point into a helper method? May be premature to do it now though.